gensrc/thrift/FrontendService.thrift

```diff
@@ -948,6 +948,7 @@ struct TTxnCommitAttachment {
     2: optional TRLTaskTxnCommitAttachment rlTaskTxnCommitAttachment
     3: optional TMiniLoadTxnCommitAttachment mlTxnCommitAttachment
     10: optional TManualLoadTxnCommitAttachment manualLoadTxnCommitAttachment
+    11: optional i64 timestampMs
```

Comment:
What does this `timestampMs` field stand for?
For a transaction, there will be so many kinds of timestamp in one transaction. You should try to rename it to help others understand.